mps-model-adapters/src/main/kotlin/org/modelix/model/mpsadapters/MPSLanguageRepository.kt

```diff
@@ -27,17 +27,20 @@ import org.modelix.model.api.ILanguageRepository
 
 class MPSLanguageRepository(private val repository: SRepository) : ILanguageRepository {
     override fun resolveConcept(uid: String): IConcept? {
-        if (!uid.startsWith("mps:")) return null
+        for (prefix in listOf("mps:", "mps-concept:")) {
```

Comment:
These prefixes seem to be repeated throughout the whole code base. Can we introduce constants for them?